src/test/apps.test.js

```diff
@@ -563,7 +563,7 @@ describe("App validator", function () {
 describe("App Page component", function () {
 
   beforeEach(function () {
-    var app = _.extend(appScheme, {
+    var app = React.addons.update(appScheme, {$merge: {
```

Comment:
I don't like this long notation of ```React.addons.update(appScheme, {$merge:```.
Could we hide this in the utils file? 
```
utils.merge(dest, src);
```
Would be cool.